pkg/apis/config/defaults.go

```diff
@@ -55,7 +54,7 @@ const (
 	DefaultMaxRevisionContainerConcurrency int64 = 1000
 )
 
-func defaultConfig() *Defaults {
+func defaultDefaultsConfig() *Defaults {
```

Comment:
Too much stutter 😬 
I'd keep the old name and just use the new name for the new one.

Reply:
But #consistency :smile: